### ISSUE TITLE
Suppress issues notifications to person who changed labels

### ISSUE THREAD

**Issue by EugeneZelenko** [llvm/llvm-project#54078]
It'll be great if automatic notifications will not be send to person who changed labels. Just to reduce e-mails clutter :-) However, I don't know how easy to archive this.

**Comment by asl** [comment]
I'm afraid this is up to GitHub as this is outside of our control...

**Comment by fhahn** [comment]
@EugeneZelenko are you still getting email notifications for issues where you only modified labels? I seem to never get notifications per email unless I comment on an issue or get mentioned.

**Comment by EugeneZelenko** [comment]
@fhahn: Yes, I still get notifications. I don't think for every kind of labels, but for majority of them. Just in case, I added myself only to `clang-tidy` team.
